module/purpose/Defender/src/Defender.cpp

```diff
@@ -122,6 +124,12 @@ namespace module::purpose {
         on<Provide<NormalDefender>>().then([this] {
             log<NUClear::DEBUG>("INITIAL");
             emit<Task>(std::make_unique<StandStill>());
+            // make the robot look forward
+            Eigen::Vector3d uPCt =
+                (Eigen::AngleAxisd(0, Eigen::Vector3d::UnitZ()) * Eigen::AngleAxisd(0, Eigen::Vector3d::UnitY()))
+                    .toRotationMatrix()
+                * Eigen::Vector3d::UnitX();
+            emit<Task>(std::make_unique<Look>(uPCt, true));
```

Comment:
going on from [Tom's comment ](https://github.com/NUbots/NUbots/pull/1335/files#r1574917410)
```suggestion
            emit<Task>(std::make_unique<Look>(Eigen::Vector3d::UnitX(), true));
```